content/docs/start/model-management/model-registry.md

```diff
@@ -0,0 +1,185 @@
+
+From the dashboard you will have an overview of all models, latest model
+versions as well stages each of the model versions is assigned to. You can also
+see which git repository for each model and get more details for it by clicking
+on the model name.
+
+Here you will see some extra information about a particular model - a
+description of the model, any labels that were assigned and particularly the
+history of all model registry actions on that selected model. For each model
+version you can also have a look at its metrics tracked by the experiment
+tracked.
+
+## Adding models
+
+Let's now train a model and add it to the model registry.
+
+We have three options how to add a model to the model registry. In this guide,
+we will be using DVCLive and add a model using Python code. This will also
+automatically save the model to DVC.
+
+We use the [`log_artifact`](/doc/dvclive/live/log_artifact) method to save the
+model and add it to the model registry. Open the training notebook
+`notebooks/TrainSegModel.ipynb` in our example repository and in the last cell
+of the notebook add the method call inside the `with Live(...)` statement as
+follows.
+
+```python
+with Live(...) as live:
+
+...
+
+    live.log_artifact(
+        path="models/model.pkl",
+        type="model",
+        name="pool-segmentation",
+        desc="This is a Computer Vision (CV) model that's segmenting out swimming pools from satellite images.",
+        labels=["cv", "segmentation", "satellite-images"],
+    )
+```
+
+Here the `path` parameter tells DVC that our model is to be found under
+`"models/model.pkl"`, the `type` parameter is `"model"` and so it will show up
+in the Studio registry (other artifact types will not) and the rest of the
+parameters are descriptive and optional and will also show up in the model
+registry.
+
+If we now run the code and commit the result to git (and push it to our git
+remote), the new model will show up in the model registry in Studio. You should
+see something like the following picture.
+
+![Newly added model in the Model Registry](/img/mr-newly-added-model.png)
+
+<details id="push-click-to-see-other-ways-to-add-models">
+
+#### 💡 Expand to see other ways to add models
+
+The other two options are to use the Studio's graphical user interface to add
+models interactively or to manually edit `dvc.yaml` files to add information
+about model artifacts. To get more details on the ways to add models have a look
+at the
+[Model registry documentation](/doc/studio/user-guide/model-registry/add-a-model).
+
+</details>
+
+## Versioning models
+
+Now that we have our first model in the model registry, we can start registering
+model versions for the model. We do it by choosing a specific commit in our
+model development history and attaching a version to it to keep an easier track
+of it. We will do that directly in the Studio UI as follows
+
+![Registering model versions](/img/mr-register-model-version.gif)
```

Comment:
Should we add tabs or details sections explaining how to do it with gto cli?

Reply:
I was wondering about that. I would mention that gto works under the hood in a details section but I would otherwise keep it out of the getting started guide. Most users will not use GTO directly.

Reply:
Studio makes sense as the default, but I worry it will turn off people if there is no option to start without first signing up for Studio, and it may give the wrong impression that Studio is required to do any of this.

Reply:
@dberenbaum Check out the latest update...I added a formulation of the sort that "we recommend you use studio because it easier and has extra features but it is not strictly necessary" plus GTO steps in "find out how it works under the hood".

But maybe the initial text can be promoted a bit so that GTO is not completely hidden in the expandable details.